Tests/TestingApps/PaywallsTester/PaywallsTester/PaywallsTester.entitlements

```diff
@@ -4,6 +4,10 @@
 <dict>
 	<key>com.apple.security.app-sandbox</key>
 	<true/>
+	<key>com.apple.security.application-groups</key>
+	<array>
+		<string>group.com.revenuecat</string>
+	</array>
```

Comment:
Not sure why this is showing up here again, something I did when setting up the stacked PRs.